micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyConnectionMetrics.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.jetty;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import org.eclipse.jetty.io.ConnectionStatistics;
+
+public class JettyConnectionMetrics implements MeterBinder {
+
+    private final ConnectionStatistics connectionStatistics;
+    private final Iterable<Tag> tags;
+
+    public JettyConnectionMetrics(ConnectionStatistics connectionStatistics, Iterable<Tag> tags) {
+        this.connectionStatistics = connectionStatistics;
+        this.tags = tags;
+    }
+
+    @Override
+    public void bindTo(MeterRegistry registry) {
+        Gauge.builder("jetty.connector.connections.current", connectionStatistics, ConnectionStatistics::getConnections)
+                .tags(tags)
+                .description("The current number of open connections")
+                .register(registry);
+        Gauge.builder("jetty.connector.connections.max", connectionStatistics, ConnectionStatistics::getConnectionsMax)
+                .tags(tags)
+                .description("The maximum number of connections")
+                .register(registry);
+        Gauge.builder("jetty.connector.connections.total", connectionStatistics, ConnectionStatistics::getConnectionsTotal)
+                .tags(tags)
+                .description("The total number of connections")
+                .register(registry);
+
+        Gauge.builder("jetty.connector.bytesReceived", connectionStatistics, ConnectionStatistics::getReceivedBytesRate)
```

Comment:
I'm wondering if using a FunctionCounter with `getReceivedBytes` wouldn't be better. We generally say don't gauge something you can count. A rate could be derived from this.

I'm also concerned about the bolded part of this description for `getReceivedBytesRate` (emphasis mine): "Total number of bytes received per second **since the last invocation of this method**". That sounds like it could lead to unexpected values if the method is invoked more than once in a step interval, which we can't guarantee won't happen, I think.